build.gradle

```diff
@@ -41,6 +43,7 @@ subprojects {
     compile group: 'org.springframework', name: 'spring-beans', version: springVersion
     // testing
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: springBootVersion
```

Comment:
ah I didn't realise the framework tests would also need the `boot-starter-test` :)